src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -509,6 +509,12 @@ int goto_instrument_parse_optionst::doit()
       return CPROVER_EXIT_SUCCESS;
     }
 
+    if(cmdline.isset("print-global-state-size"))
+    {
+      print_global_state_size(goto_model);
+      return CPROVER_EXIT_SUCCESS;
```

Comment:
Probably a stupid question but what is the use-case for this?  Is bits necessarily the most useful piece of information?

Reply:
A question that has come up is "what is the size of the state space?" (let's not discuss whether that is even a meaningful question). The output here does provide guidance: if the non-constant part of global variables is in the order of millions of bits you'll likely run into trouble (even when just a few instructions are to be executed).